src/screens/MyWorkouts/MyWorkouts.tsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
-import { Text, View } from 'react-native';
+import { View } from 'react-native';
 import { NavigatorParamList } from '../DrawerNavigator';
+import SearchBarWorkout from './WorkoutSearchBar';
+import { SearchBarWorkoutProps } from './WorkoutSearchBar';
 
 type MyWorkoutsProps = NativeStackScreenProps<NavigatorParamList, 'MyWorkouts'>;
 
 const MyWorkouts: React.FC<MyWorkoutsProps> = () => {
+  const exerciseSelected = React.useRef({ exercise: '', isOk: false });
```

Comment:
Let's get rid of this, and instead make a callback function `onSelectExercise(s: string) => void` that we can pass into the search bar. For now, just have the function `console.log` the exercise name.